src/Common/src/System/Net/Sockets/SocketErrorPal.Unix.cs

```diff
@@ -97,7 +97,7 @@ static SocketErrorPal()
             { SocketError.Disconnecting, Interop.Error.ESHUTDOWN },
             { SocketError.Fault, Interop.Error.EFAULT },
             { SocketError.HostDown, Interop.Error.EHOSTDOWN },
-            { SocketError.HostNotFound, Interop.Error.ENXIO }, // not perfect, but closest match available
+            { SocketError.HostNotFound, Interop.Error.EHOSTNOTFOUND },
```

Comment:
What about the other direction?
https://github.com/dotnet/corefx/pull/32351/files#diff-f949e77aefdf158737d872976500426eR51

Reply:
I was thinking about it but I did not come up with good answer. So I decided to keep at as it currently is e.g. "not perfect" . Quick search through Linux kernel suggests, that this truly means physical device is gone e.g. sleep/resume, usb & hotplug.

Reply:
Ok